### ISSUE TITLE
v3.12.0-beta2 Looks like RenderTexture has major API breakage

### ISSUE THREAD

**Issue by hackhat** [photonstorm/phaser#3934]
I tried to run this example https://labs.phaser.io/view.html?src=src/game%20objects/render%20texture/brute%20force.js&v=dev in dev and it fails.

 "draw" and "save" methods are not on the object, api or typescript.

I see a "drawGameObject" method instead.

**Comment by hackhat** [comment]
You don't want to keep this issue open for updating the API?

**Comment by photonstorm** [comment]
There's no need, the docs are generated from source and will be automatically updated to match the new API when 3.12 is published.

**Comment by hackhat** [comment]
ah ok, do you know where I can find the docs for master version?

**Comment by hackhat** [comment]
ah ok, thanks. i was looking for an easy url instead. No worries

**Comment by photonstorm** [comment]
It will be published to the docs url on release, but if I do it now it’ll just confuse everyone. Although maybe a beta docs url might be handy. Usually releases don’t go through betas but this one changes too much to not.

**Comment by hackhat** [comment]
> Although maybe a beta docs url might be handy.

Yes, this is what I mean. If the code is available, the api should also be available.

**Comment by photonstorm** [comment]
If someone wants to sort this out I'll link to it, but it's not a constructive use of my time on its own, as it's so trivial to build locally.

**Comment by hackhat** [comment]
Sure no problem.

**Comment by hackhat** [comment]
> You can also generate new docs locally by running npm run gen (and waiting a long time, it takes over an hour to build)

That is quite long. I think is useful to have this one somewhere online instead of building 1 hour... For example I need it now, I need to wait 1 hour, I don't think is very user friendly this approach.

**Comment by hackhat** [comment]
looks like you build and check in the rendered docs but are not matching the source code. 

Looks confusing/messy.
